### DISCUSSION TITLE
Welcome to Koboldcpp-ROCm Discussions!

### DISCUSSION THREAD

**Discussion by YellowRoseCx** [YellowRoseCx/koboldcpp-rocm#31] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AryanEmbered** [comment]
I was wondering how you'd open up the quick settings thing after you have launched the model or the UI.

Also this is by far the best option for AMD users out of everything. LM studio, Jan AI, tried everything.

Great job man, also the image in the release notes is absolutely hilarious

**Reply by YellowRoseCx** [reply]
The settings for launching the model currently can't be changed without closing KoboldCPP and reopening them. And thank you, I'm glad you like it!
